getting-started/blazor/your-first-chart-using-blazor.md

```diff
@@ -282,8 +243,6 @@ Now that the data is ready let's work on the styling, positioning, and giving yo
 
 Finally, prepare to render your first chart using the `BlazorFusionCharts` component. In your "App.js" file, replace your entire code with the code sample below, [Example](https://github.com/fusioncharts/blazor-fusioncharts#quick-start).
```

Comment:
In your "Index.razor", replace your entire code with the following code sample below.